Include/internal/pycore_debug_offsets.h

```diff
@@ -73,6 +73,7 @@ typedef struct _Py_DebugOffsets {
         uint64_t id;
         uint64_t next;
         uint64_t threads_head;
+        uint64_t threads_main;
```

Comment:
Do we have to put it in the middle? This seems like a structure that people have likely had to make their own copies of, and it's _much_ easier to version a struct if additions are at the end.

(Or maybe nobody has had to make copies of this? I'm not sure if we have for our debugger, though I expect we would, internal or otherwise.)

Reply:
> Though I guess because the structs are nested that any change spoils all the offsets anyway, so it's already going to be a massive pain for people... oh well

Yep. Is to minimize people changing these without realising